### ISSUE TITLE
Ensure feature parity with non-SDK features in old bot code

### ISSUE THREAD

**Issue by Arlodotexe** [WindowsAppCommunity/WindowsAppCommunity.Discord.ServerCompanion#3]
The MVP for the Server Companion rewrite must have feature parity with the old bot. 

While we've been hard at work over the past year planning and building our vNext architecture for SDK features, the bot features which don't depend on our database or SDK are simpler and can be implemented in tandem. 

To that end, this document reviews and tracks the various features in our "old infra" which can be implemented now without the use of our SDK. 

## Existing non-SDK features

The original TypeScript code used Discord.js, but we'll be using Remora for our implementation.

We already have an end-to-end testbed setup for developers to run the bot rewrite in our test server.
- Documentation is pending, follow-up review in another issue.
- Talk to a staff member for access, we'll set you up 1:1 for now

The commands that need consideration can be found at [uwpcommunity-backend/tree/dev/src/bot/commands](https://github.com/WindowsAppCommunity/uwpcommunity-backend/tree/dev/src/bot/commands). Each file is a command that can be run, and the default TS import declares the parameters.

As of [a75cc7ac5bd3761aef25f4aebd869cfcd23cbce4](https://github.com/WindowsAppCommunity/uwpcommunity-backend/commit/a75cc7ac5bd3761aef25f4aebd869cfcd23cbce4), these are:
- autoregister.ts
- getuser.ts
- infraction.ts
- news.ts
- ping.ts
- portal.ts
- project.ts 
- role.ts 
- staffpoll.ts
- syncroleswithbackend.ts

The only commonly used commands here are:
- infraction.ts
- portal.ts
- project.ts

Of these three commands, the **project** command is part of our SDK, so only the **infraction system** and the **portal command** can be implemented without it.

## Required work

The **infraction system** and **portal command** need to be implemented in the new bot using Remora. 
- These should use slash commands
- They should be functionally identical to the old code, except where noted.

Details and requirements will be filed in separate tickets from here. Come back soon.

**Comment by Arlodotexe** [comment]
Blocked by https://github.com/WindowsAppCommunity/WindowsAppCommunity.Discord.ServerCompanion/issues/13
